bin/replication.js

```diff
@@ -105,6 +105,49 @@ program.version(version);
         });
 });
 
+program
+    .command('multi-setup')
+    .description('NOTE: Only works for external sites')
```

Comment:
Why does it only work for external sites?

Reply:
I wasn't sure how to include another Scality site in a one-to-many configuration.
If I were to include a Scality site, would I go through the same process of a one-to-one setup to another Scality site, setting the arn to `sourceArn,targetArn` but also setting StorageClass to the target external site names (excluding the site name for the target Scality site)?

Reply:
Exactly! You would include the Scality replication endpoint name as one of the comma-separated storage classes. Then the role would be a comma-separated pair of role ARNs (i.e., `sourceArn,targetArn`) and the destination bucket ARN would be the bucket in that Scality instance to replicate to.